apps/ledger-live-mobile/src/components/SupportLinkError.tsx

```diff
@@ -3,20 +3,24 @@ import { Linking } from "react-native";
 import { Trans } from "react-i18next";
 import { urls } from "../config/urls";
 import ExternalLink from "./ExternalLink";
+import { LearnMoreLink } from "./Alert";
 
 type Props = {
   error: Error | null | undefined;
+  type?: "alert";
```

Comment:
Should it default to alert? Won't that change the default behavior?

Reply:
It's undefined or alert so it shouldn't change the default behaviour since all of them doesn't have type except the one i want